README.md

```diff
@@ -1 +1,75 @@
-# Budget_App
\ No newline at end of file
+# Budget App
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Budget APp ](#-blog-app-)
+  - [🛠 Built With ](#-built-with-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 BUDGET App <a name="about-project"></a>
+
+> "Budget App" is a mobile web application that gives users the power to manage their budget: users can have a list of expenses (e.g bed) associated with a broader category(Home) so that they can know how much they spent on what.
+
+## Built With
+- Ruby
+- Git
+- Github
+- PostgreSQL
+- JavaScript
+- Rubocop
+- RSpec
+- Stylelint
+
+
+## Getting Started
+To get a local copy of the program running, follow these simple steps
+
+
+### Prerequisites
+- IDE, preferably Visual Studio Code
+
+### Install
+- Ruby
+- Rails
+
+### Setup
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone https://github.com/shihabrafio/Budget_App
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md), please add a link to the deployed project in the README for a good README file 👍🛠.